Sources/Core/CodableReflection/Decodable+Reflectable.swift

```diff
@@ -10,7 +10,7 @@ extension Reflectable where Self: Decodable {
     /// Default `Reflectable` implementation for types that are also `Decodable`.
     ///
     /// See `AnyReflectable`.
-    public static func anyReflectProperty(valueType: Any.Type, keyPath: AnyKeyPath) throws -> ReflectedProperty? {
+    public static func anyReflectProperty<T>(valueType: T.Type, keyPath: AnyKeyPath) throws -> ReflectedProperty? {
```

Comment:
@tanner0101 this is used by Fluent, and won't work without changing it. Ideas?

Reply:
What part won't work without making that generic?

Reply:
FluentProperty.swift:
```
guard let property = try! reflectable.anyReflectProperty(valueType: valueType, keyPath: keyPath) else {
            fatalError("Could not reflect property `\(keyPath)`.")
        }
```

Reply:
Without that being generic, essentially we get the same problem, but just earlier:
```
public static func anyReflectProperty(valueType: Any.Type, keyPath: AnyKeyPath) throws -> ReflectedProperty? {
        return try anyDecodeProperty(valueType: valueType, keyPath: keyPath)
    }
```